mm2src/coins/nft/storage/mod.rs

```diff
@@ -171,3 +172,13 @@ impl<'a> NftStorageBuilder<'a> {
         sql_storage::SqliteNftStorage::new(self.ctx)
     }
 }
+
+fn get_offset_limit(max: bool, limit: usize, page_number: Option<NonZeroUsize>, total_count: usize) -> (usize, usize) {
```

Comment:
Please either make clear types or write comments on functions like this so that we can understand what is happening when we review, debug, or work on it.

Reply:
We have the same types for pagination in the whole project. 
Well I will add doc comment, that this function calculates offset and limit for final result pagination.

Reply:
Yeah, please add doc-comment.

> We have the same types for pagination in the whole project.

This should be refactored(not necessarily in this PR). https://github.com/KomodoPlatform/atomicDEX-API/issues/1883

Reply:
Done